clap_man/src/render.rs

```diff
@@ -30,8 +30,38 @@ pub(crate) fn description(roff: &mut Roff, app: &clap::App) {
 
 pub(crate) fn synopsis(roff: &mut Roff, app: &clap::App) {
     let mut line = vec![bold(app.get_name()), roman(" ")];
+    let mut items: Vec<_> = app
+        .get_arguments()
+        .filter(|i| !i.is_set(ArgSettings::Hidden))
+        .collect();
+
+    // Render the help argument
+    items
+        .iter()
+        .filter(|a| a.get_long().unwrap_or(&"default".to_string()) == "help")
+        .for_each(|opt| {
+            let (lhs, rhs) = option_markers(opt);
+            let long = opt.get_long().unwrap();
+            let short = opt.get_short().unwrap();
+            line.push(roman(lhs));
+            line.push(bold(&format!("-{}", short)));
+            line.push(roman("|"));
+            line.push(bold(&format!("--{}", long)));
+            line.push(roman(rhs));
+            line.push(roman(" "));
+        });
```

Comment:
Let's start out focusing solely on the users defined sort order rather than overriding it.  Anything past that should be a separate issue for us to discuss and resolve.

Reply:
I have fixed this as suggested, now sorting by `display order` and then alphabetically for remaining arguments.